# Get DC Motor Configuration Service
uint8 motor_id
---
bool success
DCMotorConfig config
string message